### ISSUE TITLE
Fix installation on Windows and update Readme

### ISSUE THREAD

**Issue by danielhers** [hasadna/anyway#174]
Several people have tried running the app on Windows unsuccessfully, and eventually decided to install a Linux VM instead.
We want it to work on Windows so everyone can develop easily. The failure point seems to be the `pip install -r requirements`: some packages fail to install and we haven't found a way to fix that yet.

**Comment by aviklai** [comment]
I succeeded to install anway in windows. There are 2 packeged that can't be installed on windows:
1. pyproj==1.9.4
2. psycopg2==2.5.5

both of them need to be downloaded from:
http://www.lfd.uci.edu/~gohlke/pythonlibs/
(As the readme states) using the cp27 win32 version, even if you have a 64 bit windows.

Another issue I encountered is with the sqlite database.
for some reason, the command:
set DATABASE_URL="sqlite:///local.db"
doesn't work correctly because afterwards when I run models.py, I get an error.

The solution I found is to replace in config.py the SQLALCHEMY_DATABASE_URI variable to:
SQLALCHEMY_DATABASE_URI = os.path.join('sqlite:////', 'local.db')

Then when I run models.py, the local.db is created in c:\ folder and the app and then we can rus process.py and main.py and it works.
